exporter/otlpexporter/internal/arrow/exporter_test.go

```diff
@@ -0,0 +1,341 @@
+package arrow
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"testing"
+	"time"
+
+	"go.opentelemetry.io/collector/component/componenttest"
+	"go.opentelemetry.io/collector/internal/testdata"
+
+	arrowpb "github.com/f5/otel-arrow-adapter/api/collector/arrow/v1"
+	arrowMock "github.com/f5/otel-arrow-adapter/api/collector/arrow/v1/mock"
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zaptest"
+	"google.golang.org/grpc"
+)
+
+var (
+	waitForReadyOption = []grpc.CallOption{
+		grpc.WaitForReady(true),
+	}
+
+	nowaitForReadyOption = []grpc.CallOption{
+		grpc.WaitForReady(true),
```

Comment:
Shouldn't that be false in this case?

Reply:
Yes. Fixed and commented out. This was unused, presently.